### ISSUE TITLE
Form in Currencies Sidebox Missing Aria Label (Accessibility)

### ISSUE THREAD

**Issue by dbltoe** [zencart/zencart#6016]
Apologies for not catching this in one of our earlier sweeps but the currencies sidebox fails standard 4.1.2 Name, Role, Value.

That means the standard 
`$content .= '<label for="currency">Select Currency:</label>';`
will not clear the failure.

It needs a select and option field that I have not been successful in creating myself or I would submit it for consideration.

**Comment by OldNGreyBMR** [comment]
I have addressed this issue with the following code. It may not be the neatest solution but it works. The production site (zc158) is bmhstore DOT bmh DOT com DOT au

Line 3 and line 8 are additional

```php
$content = '';
$content .= '<div id="' . str_replace('_', '-', $box_id . 'Content') . '" class="sideBoxContent centeredContent">';
$content .= '<label for= "select-currency" >';
$content .= zen_draw_form('currencies_form', zen_href_link(basename(preg_replace('/.php/','', $PHP_SELF)), '', $request_type, false), 'get');
$content .= zen_draw_pull_down_menu('currency', $currencies_array, $_SESSION['currency']) . $hidden_get_variables . zen_hide_session_id();
$content .= zen_image_submit(BUTTON_IMAGE_SUBMIT, BUTTON_SUBMIT_GO_ALT);
$content .= '</form>';
$content .= '</label>';
$content .= '</div>';
``` 

OldNGrey

**Comment by scottcwilson** [comment]
@OldNGreyBMR Would you please PR?

**Comment by OldNGreyBMR** [comment]
Totally new to this. 
In the doc "https://docs.zen-cart.com/dev/contributing/github_workflow/" under the heading "Making a workin-branch ...", under bullet point "SourceTree" it has
```
First, make sure you’re in “History View” (View, History View)
Find where it shows **upstream/v158**, and right-click on that row. Choose Branch… from the pop-up menu
```
Should I still use "upstream/v158" or else there is a tag " V2.0.0-alpha1"

**Comment by scottcwilson** [comment]
Thanks - I will fix this.  You want to checkout against the master branch.

**Comment by OldNGreyBMR** [comment]
Scott
Thanks
I managed to work it out and did a checkout against master branch.
Lat9 responded to the change.
Bailey
